SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMigration/SpatialTestCharacterMigration.cpp

```diff
@@ -38,19 +39,25 @@ ASpatialTestCharacterMigration::ASpatialTestCharacterMigration()
 	Author = "Victoria";
 	Description = TEXT("Test Character Migration");
 	TimeLimit = 300;
+
+	PositionOnServerOne = FVector(-200.0f, 0.0f, 40.0f);
+	PositionOnServerTwo = FVector(200.0f, 0.0f, 40.0f);
 }
 
 void ASpatialTestCharacterMigration::PrepareTest()
 {
 	Super::PrepareTest();
 
-	// Reset test
-	FSpatialFunctionalTestStepDefinition ResetStepDefinition(/*bIsNativeDefinition*/ true);
-	ResetStepDefinition.StepName = TEXT("Reset");
-	ResetStepDefinition.TimeLimit = 0.0f;
-	ResetStepDefinition.NativeStartEvent.BindLambda([this]() {
-		bCharacterReachedDestination = false;
-		bCharacterReachedOrigin = false;
+	// Wait for actor to be stationary
+	FSpatialFunctionalTestStepDefinition WaitForStationaryActorStepDefinition(/*bIsNativeDefinition*/ true);
+	WaitForStationaryActorStepDefinition.StepName = TEXT("WaitForStationaryActorStepDefinition");
+	WaitForStationaryActorStepDefinition.NativeTickEvent.BindLambda([this](float DeltaTime) {
+		const UGameInstance* GameInstance = GetGameInstance();
+		const FString WorkerId = GameInstance->GetSpatialWorkerId();
```

Comment:
nit, could also just do `GetGameInstance()->GetSpatialWorkerId()` if we don't expect the game instance to be null, but nbd